### ISSUE TITLE
[pipeline] Don't need to specify the annotators? 

### ISSUE THREAD

**Issue by danyaljj** [CogComp/cogcomp-nlp#418]
Now that loading the models in annotators is lazy, I suppose we don't need these if-elses? 
https://github.com/CogComp/cogcomp-nlp/blob/master/pipeline/src/main/java/edu/illinois/cs/cogcomp/pipeline/main/PipelineFactory.java#L183-L284

**Comment by mssammon** [comment]
This is true for some annotators, but not others (Stanford components are not lazily loaded, and possibly some CCG annotators)

**Comment by danyaljj** [comment]
Relevant to this why can't we make our dependency and Stanford components lazy? 
https://github.com/CogComp/cogcomp-nlp/blob/master/depparse/src/main/java/edu/illinois/cs/cogcomp/depparse/DepAnnotator.java#L37-L42
